DataAccess/Scripts/insert_users_profiles.sql

```diff
@@ -1,6 +1,7 @@
 use BandIt;
```

Comment:
Add rows into UserSkills. Right now, after running this script, all users have no skills. That's kinda sad :(